Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM node:12
+FROM node:16
 WORKDIR /usr/src/app
+# Bundle app source
 COPY package*.json ./
 RUN npm ci --only=production
-# Bundle app source
+# COPY everything not in dockerignore file
 COPY . .
-ENTRYPOINT [ "node", "bin/load.js" ]
+# set to avoid errors when singularity overloads working dir
+ENV NODE_PATH=/usr/src/app/node_modules
```

Comment:
is the absolute path here going to be a problem if a user's path is different?

Reply:
no b/c this is the path inside the docker container